VirtoCommerce.Platform.Security/Authorization/PermissionAuthorizationPolicyProvider.cs

```diff
@@ -17,7 +17,7 @@ public class PermissionAuthorizationPolicyProvider : DefaultAuthorizationPolicyP
         private readonly IConfiguration _configuration;
         private readonly IPermissionsRegistrar _permissionsProvider;
         private readonly IPlatformMemoryCache _memoryCache;
-        public PermissionAuthorizationPolicyProvider(IOptions<AuthorizationOptions> options, IConfiguration configuration, IPermissionsRegistrar permissionsProvider, IPlatformMemoryCache memoryCache)
+        public PermissionAuthorizationPolicyProvider(IOptions<Microsoft.AspNetCore.Authorization.AuthorizationOptions> options, IConfiguration configuration, IPermissionsRegistrar permissionsProvider, IPlatformMemoryCache memoryCache)
```

Comment:
I think that fully-qualified class names in the code are hard to read and understand. IMO, it's better to use aliases, like this:
```csharp
using AspNetCoreAuthorizationOptions = Microsoft.AspNetCore.Authorization.AuthorizationOptions;
// ...
public PermissionAuthorizationPolicyProvider(IOptions<ASpNetCoreAuthorizationOptions> options, ...)
```